examples/browserify/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
This .gitignore affects only browserify example folder, so it does touch anything on the outside. Or there is something else you're thinking about?